modules/installation-creating-aws-worker.adoc

```diff
@@ -126,6 +126,13 @@ the CloudFormation template for the security group and roles.
 * `m5a.8xlarge`
 * `m5a.12xlarge`
 * `m5a.16xlarge`
+* `m6i.large`
```

Comment:
there are no limitations in UPI templates anymore, https://github.com/openshift/installer/pull/5166

Reply:
@yunjiang29 so to keep consistency, and simplicity, we should remove all that instance types (legend 22) to refer to the section "Supported AWS machine types", right?

Reply:
@mtulio this section is the description of InstanceType parameter in UPI templates,  I do not think it should refer to any other section, b/c this is the sample templates for UPI installation, users can choose any types to base their needs

Reply:
> @mtulio this section is the description of InstanceType parameter in UPI templates, I do not think it should refer to any other section, b/c this is the sample templates for UPI installation, users can choose any types to base their needs

@yunjiang29 Ok, but keeping an long outdated list as a sample can be very confusing for customers. I suggest to update with supported instance sizes or just remove the entire list and keep generic values like: `InstanceType.large` (workers) and `InstanceType.xlarge` for masters. I will remove the change from this PR to not block on non-mandatory sections.

Reply:
@mtulio I suggest to remove the entire list to keep it consistent as UPI template, WDYT?

Reply:
> @mtulio I suggest to remove the entire list to keep it consistent as UPI template, WDYT?

@yunjiang29 perfect. Updated. Please let me know if it looks better.